src/nile/cli.py

```diff
@@ -103,15 +120,18 @@ def declare(
         alias=alias,
         max_fee=max_fee,
         overriding_path=overriding_path,
+        watch_mode=watch_mode,
     )
 
 
 @cli.command()
 @click.argument("signer", nargs=1)
 @network_option
-def setup(signer, network):
+@click.option("--track", "-t", "watch_mode", flag_value="track")
+@click.option("--debug", "-d", "watch_mode", flag_value="debug")
+def setup(signer, network, watch_mode):
     """Set up an Account contract."""
-    Account(signer, network)
```

Comment:
This is probably for a different issue, but I think we should refactor the account to avoid deploying when the instance is being initialized and make the deploy step explicit by calling deploy if necessary. I think this would make the code clearer.